components/automate-gateway/api/compliance/reporting/stats/stats.proto

```diff
@@ -122,10 +122,14 @@ message NodeSummary {
 }
 
 message Stats {
-	int64	nodes = 1;
-	int32 platforms = 2;
-	int32 environments = 3;
-	int32 profiles = 4;
+	int32 nodes_cnt = 1;
+	//int64 types render into string types when serialized to satisfy all browsers
+	//we don't really need for nodes counts to be int64 as int32 limits us to 2billion nodes which is plenty for now
+	//we are therefore deprecating nodes and favor nodesCnt
+	int64 nodes = 2 [deprecated = true];
+	int32 platforms = 3;
+	int32 environments = 4;
+	int32 profiles = 5;
```

Comment:
Changing the IDs here is technically a breaking API change.  Because of how we build it probably don't matter, but the "protobufy" way to do this would be to leave all the existing IDs as is and add nodes_cnt to the end.

Reply:
does that really matter?  who uses the ID?

Reply:
The IDs are used in the serialization of protobuf message on the wire.  So, if you want to be able to receive messages from a client compiled with the previous version of your protobuf, you need to keep the IDs consistent.

Reply:
ok, changing now. thanks

Reply:
FWIW, I believe that changing the numeric type of an item (i.e. `int64` -> `int32`) is "compatible" in the wire-format sense so you can sometimes get away with just changing one numeric type to another, but I haven't thought about it hard enough to know if that would be OK here.

Reply:
> FWIW, I believe that changing the numeric type of an item (i.e. `int64` -> `int32`) is "compatible" in the wire-format s.. 
 
yeah, the problem is that for API users that are deserializing the json and using the nodes attribute that is currently of type string (because that's what it does to int64), and doing whatever subsequent type conversion to turn it to a number, we would break them if we just changed this to int32 because it would change the json representation to a number